haddock-api/src/Haddock/Backends/Xhtml.hs

```diff
@@ -412,6 +471,14 @@ ppJsonIndex odir maybe_source_url maybe_wiki_url unicode pkg qual_opt ifaces = d
 
     links_info = (maybe_source_url, maybe_wiki_url)
 
+    -- update link using relative path to output directory
+    fixLink :: FilePath
+            -> JsonIndexEntry -> JsonIndexEntry
+    fixLink ifaceFile jie = 
+      jie { jieLink = makeRelative odir (takeDirectory ifaceFile)
+                        FilePath.</> jieLink jie }
+
+
```

Comment:
Extraneous blank line here

Reply:
I think that `haddock` is using two empty lines between functions, are you sure you'd like to remove it?

Reply:
If only. :) 
But the style is quite inconsistent, one or two lines, so I'm tying to unify all that.

Reply:
ok, I'll remove it.